packages/zoe/src/contractSupport/atomicTransfer.js

```diff
@@ -0,0 +1,79 @@
+const { details: X, quote: q } = assert;
+
+/**
+ * @typedef {[
+ *   fromSeat?: ZCFSeat,
+ *   toSeat?: ZCFSeat,
+ *   subtrahend?: AmountKeywordRecord,
+ *   addend?: AmountKeywordRecord
+ * ]} TransferArgs
+ */
```

Comment:
nit: this type is the concern of `atomicTransfer` so better to be defined there. To have a name for the type of the second arg (I see it's imported elsewhere):

```suggestion
/** @typedef {Parameters<typeof atomicTransfer>[1]} TransferArgs */
```

Reply:
This seems less clear to me.

Reply:
Btw, I also got rid of the awful names `subtrahend` and `addend`, replacing with `fromAmounts` and `toAmounts`.

Reply:
Resolving